modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -999,27 +999,34 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
      class CV_EXPORTS_W_SIMPLE Model : public Net
      {
      public:
-         Model(const std::string& model, const std::string& config = "", const Size& size = {-1, -1},
-               const Scalar& mean = {}, float scale = 1.0, bool swapRB = true, bool crop = false);
-
-         Model(const Net& network, const Size& size = {-1, -1},
-               const Scalar& mean = {}, float scale = 1.0, bool swapRB = true, bool crop = false);
+         Model(const std::string& model, const std::string& config = "");
+         Model(const Net& network);
 
          Model& setInputSize(const Size& size);
+         Model& setInputSize(int height, int width);
```

Comment:
That might be the source of confusion because `cv::Size` has constructor with `width, height` order: https://docs.opencv.org/master/d6/d50/classcv_1_1Size__.html#a45c97e9a4930d73fde11c2acc5f371ac